extensions/sql-bindings/src/common/utils.ts

```diff
@@ -162,6 +162,18 @@ export function escapeClosingBrackets(str: string): string {
 	return str.replace(']', ']]');
 }
 
+export function checkBrackets(objectName: string): string {
+	if (objectName.includes('.')) {
```

Comment:
Object names can have . in them too so this check isn't enough. (e.g. a table named `My.Table` is valid).